code/controllers/subsystem.dm

```diff
@@ -2,6 +2,7 @@
 /datum/controller/subsystem
 	// Metadata; you should define these.
 	name = "fire coderbus" //name of the subsystem
+	var/ss_id = "fire_coderbus_again"
```

Comment:
You currently only set this on the timer and processing subsystems, not regular subsystems.